test/unit/chat/__snapshots__/bookmarks.test.ts.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`findBookmarkDocument complains if you have multiple bookmark docs 1`] = `"More than one bookmark file! <http://example.com/bookmarks2.ttl>, <http://example.com/bookmarks3.ttl>, <http://example.com/bookmarks1.ttl>"`;
+exports[`findBookmarkDocument complains if you have multiple bookmark docs 1`] = `"More than one bookmark file! <http://example.com/bookmarks2.ttl>,<http://example.com/bookmarks3.ttl>"`;
```

Comment:
```suggestion
exports[`findBookmarkDocument complains if you have multiple bookmark docs 1`] = `"More than one bookmark file! <http://example.com/bookmarks2.ttl>, <http://example.com/bookmarks3.ttl>"`;
```

Reply:
This appears to be a human-facing message. If that's correct, then the space I tried to add is appropriate. If that's not correct, then it should indeed be left out, but a comment to that effect would be appropriate here.